docs/blog/posts/using-vega-protocol-with-hummingbot/index.md

```diff
@@ -0,0 +1,172 @@
+---
+date: 2024-02-09
+authors:
+  - foundation
+categories:
+  - Guides
+---
+
+# User Guide: Using Vega Protocol with Hummingbot
+
+![cover](cover.webp)
+
+
+## Introduction
+
+Welcome to our user tutorial on integrating the power of Vega Protocol with Hummingbot for your trading journey. In this guide, we'll walk you through the detailed steps required to seamlessly set up a Vega Wallet, link it with Hummingbot, and embark on your trading adventures. 
+
+Whether you're new to automated trading or looking to leverage Vega Protocol's decentralized trading capabilities, this guide is designed to provide you with all the information you need to get started. Let's dive into the world of blockchain-based trading with Vega Protocol and Hummingbot, and unlock new possibilities in your trading strategy.
+
+<!-- more -->
+
+## Prerequisites
+
+### Create a Vega Wallet
+
+A Vega Wallet is essential for interacting with Vega, whether it's for staking or trading. The Vega Wallet apps allow you to manage wallets and key pairs, deposit and withdraw assets, stake and sign transactions. There are several options to create a Vega Wallet app including Chrome / Firefox browser extensions, or a desktop / CLI app. Or if you have an existing Metamask account you can setup a Vega Key through Metamask Snaps which we recommend below. 
+
+
+**Browser Extension**
+
+- [Chrome extension:](https://chrome.google.com/webstore/detail/vega-wallet-mainnet/codfcglpplgmmlokgilfkpcjnmkbfiel)
+
+- [Firefox extension:](https://addons.mozilla.org/en-GB/firefox/addon/vega-wallet-mainnet/)
+
+
+**Desktop Wallet**
+
+- [Download the app:](https://github.com/vegaprotocol/vegawallet-desktop/releases) On GitHub, download the desktop app for your operating system.
+
+- [Get started:](https://docs.vega.xyz/mainnet/tools/vega-wallet/desktop-app/getting-started) Read a guide on using the desktop app to connect an existing Vega wallet, create or restore a wallet and start interacting with your keys.
+
+
+**Metamask Snap** (recommended)
```

Comment:
Make this the first option since that's what most people will do:

1. Use Metamask Snap (recommended)
2. Download Browser Extension
3. Download Desktop Wallet